src/reporter.rs

```diff
@@ -27,7 +28,6 @@ impl JaegerCompactReporter {
     }
 
     /// Sets the address of the report destination agent to `addr`.
-    ///
```

Comment:
Is this an intended change?
If not, please revert the diff.

Reply:
my bad, this wasn't intended. I've pushed the change.